steppy/base.py

```diff
@@ -302,10 +311,10 @@ def fit_transform(self, data):
             dict: Step outputs from the ``self.transformer.fit_transform`` method
         """
         if self.output_is_cached and not self.force_fitting:
-            logger.info('Step {} loading cached output...'.format(self.name))
-            step_output_data = self._load_output(self.exp_dir_cache_step)
+            logger.info('Step {} using cached output'.format(self.name))
+            step_output_data = self.output
         elif self.output_is_persisted and self.load_persisted_output and not self.force_fitting:
```

Comment:
@kamil-kaczmarek if output is both cached and persisted it will load it twice I think. So probably it should be simply solved with a bunch of ifs

Reply:
we have `if, elif, else`, so if output is cached step will load cached output and will not evaluate elif else statements.